src/Controls/tests/TestCases.HostApp/Issues/Issue25224.xaml.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Windows.Input;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+	[Issue(IssueTracker.Github, 25224, "CollectionView - EmptyView with EmptyViewTemplate for Data template selector page throws an exception", PlatformAffected.iOS)]
+	public partial class Issue25224
+	{
+		public Issue25224()
+		{
+			InitializeComponent();
+			BindingContext = new Issue25224ViewModel();
+		}
+	}
+
+	public class SearchTermDataTemplateSelector : DataTemplateSelector
```

Comment:
Can you move all of these classes inside the scope of the `class Issue25224` class? This way they aren't part of the `Maui.Controls.Sample.Issues` namespace?